server/app.py

```diff
@@ -60,6 +60,37 @@ def createAccount():
 
     return jsonify(message="Successfully updated resource."), 200
 
+# Login
+@app.route("/api/Login", methods=["POST"])
+def Login():
+    username=request.get_json()["username"]
+    password=request.get_json()["password"]
+   
+    print('username', username)
+    print('password', password)
+
+    config = {
+        "user": "root",
+        "password": "sherlockeD123",
+        "host": "db",
+        "port": "3306",
+        "database": "MSCI",
+    }
+
+    connection = mysql.connector.connect(**config)
+    cursor = connection.cursor()
+
+    query = "SELECT * FROM Users WHERE userName = %s AND userPassword = %s;"
+
+    cursor.execute(query, [username, password])
+
+    results = cursor.fetchone()
+
+    if results:
+        return jsonify("Successful Login"), 200
+    else:
+        return jsonify("Invalid Username or Password"), 401
```

Comment:
```suggestion
    with DbSelector() as d:

         query = "SELECT * FROM Users WHERE userName = %s AND userPassword = %s;"
         
         cursor.execute(query, [username, password])
         
         results = cursor.fetchone()

    if results:
        return jsonify("Successful Login"), 200
    else:
        return jsonify("Invalid Username or Password"), 401
```

Reply:
heres an example of how it might look. I haven't tested it, its just to show how we've been doing this

Reply:
I won't accept the suggestion just because we have not tested it yet. However since this is a minor detail that we have not previously discussed as a team I would consider it negligible to our acceptance criteria. I am comfortable approving this PR and discussing this at our next team meeting.

Reply:
with DbSelector() as d:
        query = "SELECT * FROM Users WHERE userName = %s AND userPassword = %s;"

        d.cursor.execute(query, [username, password])

        results = d.cursor.fetchone()

        if results:
            return jsonify("Successful Login"), 200
        else:
            return jsonify("Invalid Username or Password"), 401

Reply:
@m2qaderi  I edited the above suggested code block! so the code above is correct and working